Unbound.cs

```diff
@@ -178,15 +178,28 @@ void Start()
             defaultCards = CardChoice.instance.cards;
             activeCards.AddRange(defaultCards);
             
-            // add default levels to level list
-            levels.AddRange(MapManager.instance.levels);
 
             // register default cards with toggle menu
             foreach (var card in defaultCards)
             {
                 CardToggleMenuHandler.Instance.AddCardToggle(card, false);
             }
 
+            // add default levels to level list
+            levels.AddRange(MapManager.instance.levels);
+
+            // add custom levels to map list
+            this.ExecuteAfterSeconds(0.1f, () =>
+            {
+                MapManager.instance.levels = levels.ToArray();
+                On.MainMenuHandler.Awake += (orig, self) =>
```

Comment:
Registering a hook after waiting 0.1f seconds doesn't quite make sense. I'm not sure if using an MMHOOK here is necessary at all.